### ISSUE TITLE
A few Questions

### ISSUE THREAD

**Issue by bjggi** [Mirdain/Silmaril#1]
First of all, thanks for your work, can't wait to finally replace good old CP :)

I 6-Box since a long time and would love to test this even more.

**Questions**
- Would it be possible to add an option (checkbox), so the window isn't always on top?
_**This has been implemented - next update**_
- What ingame commands are there? And is it possible to Start/Pause Silmaril with a command? Would love to be able to use ingame command to Start/Stop like CP.
_**This is currently available and I posted the command in the discord server and will try to update github**_

- Would it be possible to upload Default Job XML (WHM & GEO i would be interested) in the future, I think your experience regarding the settings would certainly be helpful at the beginning
 _**I do plan to do this.  Just been busy with initial launch and updates**_
- I need to load the Addon on all my Chars, even if i don't use it on that specific Char, right?
_**Correct**_

**GEO**
- If I'm reading this correctly, it isn't possible to enfeeble yet? I.e. GEO cannot yet cast a spell to put itself on the hate list (Dia II/Dispelga or Cure on a Player) after it places a debuff bubble right?
_**Enfeebling not done yet but under combat script you can do a /ma "Dia II" and it will cast on every mob it is assisting.**_

**WHM**
- Somehow my WHM doesn't cast Barparalyzra, all other Barspells are working.
 _**Spelling and a target mistake - found last night and fixed for next upate**_

- When someone new comes into the party, do you always have to put them on the cure/debuff list? Like if a Friend joins the Party, not one of your alts, or is this only if u want to prioritize and set a specific sets of debuff for a player?
_**Everytime - there was an issue with enhancing.  I'm trying to think of a good way to do this.  Might have option**_

That's it for now, i hope its ok to post Feedback here. If there will be a Command to Start/Stop i will gladly test this on real events :)

Best regards and thx again!

**Comment by bjggi** [comment]
Thx @Mirdain  for your answers, looking forward to the update!

_**Every time - there was an issue with enhancing. I'm trying to think of a good way to do this. Might have option**_

Sounds good, maybe it's possible to add like a "Default" that gets used for all Party Members if not else specified with the "name" of the Party member.

If i may add something else for the future :) 

Peach did something very interesting with her Healbot https://github.com/PeachBlossomWine/HealBot

She added a [Aura Handling](https://github.com/PeachBlossomWine/HealBot/commit/e017abd15974db35cb23d60dcf958772e18b1d16), for stuff like Odyssey or W3 Dyna bosses. Maybe that's something that could be added to Silmaril. So the WHM doesn't try to dispel something it can't. Would also reduce the numbers of XML you would have to save/load for some bosses.

Another thing she added was the possibility to add Mobs to Debufflist. You can add "Names" of Mobs that get specific debuffs. Like Odyssey where u want certain debuffs on the "Boss", but Bind/Gravity II on the "Adds". Think that's quite interesting for certain situations and helps with multiboxing.

Just some more ideas =)

**Comment by Mirdain** [comment]
From what I can tell it disables those debuffs for the entire zone.

I'd rather do it via packets and I already have a map that convert those debuffs to "geomancy" equivalent debuffs.

This allows silmaril to see them but not act on them.  I have the hooks in place but spent last night working on the auto raise.

Worse case would be Mob ID and distance check from them since very few mobs can do geomancy.

Those are things I do want to address but they are also not immediately solved b/c they are not easy :)

I do feel I will address those concerns in time but for now I want to focus on the features currently released.

**Comment by Riplinian** [comment]
Hey. Great Work and great stuff. Thanks for making this, its awesome.

Few questions. The options not currently available, greyed out, like combat, assist(works on combatscripts)  , weaponskills etc; they are to be implemented later, or am I doing it wrong?
_**For now it is disabled and you cannot enable it.**_

Tested some 1.2 version today, what i encountered:
- Movement stops on followers when the character they follow engage, following resumes when disengaging.
  _**I implemented this because it allows for the combat distances to work.  I will have to test with a locked version to see how that plays out.**_
- Combatscript delay, always instant, delay dont seem to apply, and doesnt work all the time.
_**I will look into it.  I had to move this and have not tested after the change.**_
- I cant get //sm load to work. //sm start, stop works great. Maybe I'm saving the jobfiles wrong, or missing some text?
_**Unfortunately its a syntax issue.  I will try to get a better vid on how to do this and use ingame macros if you wish.  Folder structure is important**_
